src/JsonTxBuilderBase.sol

```diff
@@ -51,4 +52,95 @@ abstract contract JsonTxBuilderBase is CommonBase {
 
         return calls;
     }
+
+    /// @notice Reads all account and storage accesses and makes a series of basic sanity checks on them.
+    ///         This function can be overridden to provide more specific checks, but should still call
+    ///         super.checkStateDiff().
+    function checkStateDiff(Vm.AccountAccess[] memory accountAccesses) internal view virtual {
+        require(accountAccesses.length > 0, "No account accesses");
```

Comment:
Any objections to adding this log back? It's nice to be able to read through the logs to make sure all expected assertions ran
```suggestion
        console.log("Running assertions on the state diff");
        require(accountAccesses.length > 0, "No account accesses");
```